src/diffusers/models/autoencoders/autoencoder_kl_cogvideox.py

```diff
@@ -172,7 +162,12 @@ def __init__(
         self.conv_y = CogVideoXCausalConv3d(zq_channels, f_channels, kernel_size=1, stride=1)
         self.conv_b = CogVideoXCausalConv3d(zq_channels, f_channels, kernel_size=1, stride=1)
 
-    def forward(self, f: torch.Tensor, zq: torch.Tensor) -> torch.Tensor:
+    def forward(
+        self, f: torch.Tensor, zq: torch.Tensor, conv_cache: Optional[List[torch.Tensor]] = None
+    ) -> torch.Tensor:
+        if conv_cache is None:
+            conv_cache = [None] * 2
```

Comment:
Sorry this is just for the time being. It is effectively the number of children modules that are directly a CausalConv3D or have a CausalConv3D. In the first iteration of the frame-wise decoding, the values passed to the `conv_cache` layer across all blocks needs to be None. Doing this just makes it a bit simple to handle. For second to n'th iteration, the conv_cache will be populated with previous temporal slice. Since the SpatialNorm3D layer has 2 CausalConv3d layers, we create a list of 2 elements here. This leads to a little cleaner implementation where we can just pop stuff instead of maintaining conditional statements.

Reply:
Ah cool. Perhaps this could be added as a comment.